apps/core/logging.py

```
@@ -3,12 +3,19 @@
+from typing import Any
 
 import requests
 
 
 class SeqHandler(logging.Handler):
-    def __init__(self, server_url, api_key=None, batch_size=1, static_fields=None):
+    def __init__(
+        self,
+        server_url: str,
+        api_key: str | None = None,
+        batch_size: int = 1,
```

Ruff (0.14.14)

[warning] Unused method argument: `batch_size`

(ARG002)